packages/core/src/wrappers/Intent.ts

```diff
@@ -0,0 +1,40 @@
+import Base from './Base'
+import { ConnectorInterface } from '../ConnectorTypes'
+import TransactionPath from '../transactions/TransactionPath'
+import TransactionRequest from '../transactions/TransactionRequest'
+
+export interface IntentData {
+  contractAddress: string
```

Comment:
I also change the `appAddress` for `contractAddress` as the intent could be for an external contract as well.